webview/src/shared/components/contextMenu/ContextMenu.tsx

```diff
@@ -66,6 +66,8 @@ export const ContextMenu: React.FC<ContextMenuProps> = ({
     onShow={onShow}
     disabled={!content || disabled}
     appendTo={'parent'}
+    followCursor={'initial'}
+    offset={[0, 0]}
```

Comment:
By using `followCursor` and `offset`, we can make our context menu behave pretty similarly to vscode context menus 🤔 

<img width="309" alt="image" src="https://user-images.githubusercontent.com/43496356/189775856-cb080192-5f3d-488e-8023-49ca00cbf4ed.png">

Though can be taken off if we prefer the original :)

Reply:
👍🏻 

Can you take a look at the storybook to see if there is anything that can be done?